docarray/array/document.py

```diff
@@ -52,6 +52,16 @@ def __new__(
         """Create a AnnLite-powered DocumentArray object."""
         ...
 
+    @overload
+    def __new__(
+        cls,
+        _docs: Optional['DocumentArrayElastic'] = None,
```

Comment:
`DocumentArraySourceType` as the type of `_docs`

Reply:
done